gallery/src/guidelines/ColorPalette/Swatcher.scss

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '~@sonatype/react-shared-components/scss-shared/nx-colors';
```

Comment:
Redundant; `nx-variables` imports this

Reply:
> Redundant; `nx-variables` imports this

Ooops: https://github.com/sonatype/sonatype-react-shared-components/pull/313/commits/2add318c9ea7af6b13b90cacc20d0452876b0014